src/test_report.py

```diff
@@ -18,9 +22,97 @@ class TestReport:
 
     def __init__(self, configs, args):
         self._db_config = configs['db_configs'][args.db_config]
+        self._email_config = configs['email_configs'][args.email_config]
         api_token = os.getenv('API_TOKEN')
         self._db = kernelci.data.get_db(self._db_config, api_token)
 
+    def create_email(self, template):
+        """Create email"""
+
+        if template:
+            email_msg = email.mime.text.MIMEText(template, "plain", "utf-8")
+
+        if self._email_config['headers']:
+            for key, val in self._email_config['headers'].iteritems():
+                val = to_unicode(val)
+                if is_ascii(val):
+                    email_msg[key] = str(val)
+                else:
+                    email_msg[key] = Header(val, "utf-8").encode()
```

Comment:
This seems like some old Python2 code. `str` in Python 3 is actually storing UTF-8 data. There is no need to convert thigs.